README.md

```diff
@@ -353,25 +353,25 @@ values loaded from the `.env` file.
+| `DATABASE__PASSWORD`                          | The database password to use for the connection string for the `MongoClient` to connect to the database.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                | Yes                       |                                                       |
+| `DATABASE__HOST_AND_OPTIONS`                  | The host (and optional port number) component as well specific options (if any) to use for the connection string for the `MongoClient` to connect to the database. The host component is the name or IP address of the host where the `mongod` instance is running, whereas the options are `<name>=<value>` pairs (i.e. `?authMechanism=SCRAM-SHA-256&authSource=admin`) specific to the connection.<br> <ul><li>For a replica set `mongod` instance(s), specify the hostname(s) and any options as listed in the replica set configuration - `prod-mongodb-1:27017,prod-mongodb-2:27017,prod-mongodb-3:27017/?authMechanism=SCRAM-SHA-256&authSource=admin`</li><li>For a standalone `mongod` instance, specify the hostname and any options - `prod-mongodb:27017/?authMechanism=SCRAM-SHA-256&authSource=admin`</li></ul>. If specified, only the value of `readPreference=primary` should be used. | Yes                       |                                                       |
```

Comment:
Idk where you intended for this to go but if it was below the bullet points then this is how it looks when rendered (note the full stop at the beginning).
![image](https://github.com/user-attachments/assets/dd10250f-d960-47d9-9517-c7717c8da293)

Reply:
Ah yeah I meant it at the end of the main text. Will fix it now.